.vscode/settings.json

```diff
@@ -1,11 +1,13 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
+    80
+  ],
   "files.exclude": {
     "**/.git": true,
     "**/node_modules": true,
     "**/build": true
   },
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
still? 😅

Reply:
Argh! Sorry - I can't do anything with that enabled and I haven't figured out how to override it in my env. Every time I save a file, something reformats it wrong and I get pretty errors.

Reply:
That's odd! I don't use vs code, so can't help you. @thymikee ideas on what's wrong?